firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -60,6 +60,32 @@ class DictionaryEntryDetailSerializer(serializers.HyperlinkedModelSerializer):
     )
     categories = CategorySerializer(many=True)
     site = SiteSummarySerializer()
+    base_characters = serializers.SerializerMethodField()
+    words_in_phrase = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_base_characters(entry):
+        if "⚑" in entry.custom_order:
+            return "Due to this entry containing unknown characters, it cannot be used in games."
+        else:
+            alphabet = Alphabet.objects.filter(site=entry.site).first()
+            cs = alphabet.sorter
+            character_list = cs.word_as_chars(entry.title)
+            for i, char in enumerate(character_list):
+                character_list[i] = alphabet.presort_transducer(char).output_string
+
+            return character_list
+
+    @staticmethod
+    def get_words_in_phrase(entry):
+        if entry.type != "PHRASE":
+            return "This entry is not a phrase."
+        else:
+            alphabet = Alphabet.objects.filter(site=entry.site).first()
+            word_list = entry.title.split(" ")
+            for i, word in enumerate(word_list):
+                word_list[i] = alphabet.presort_transducer(word).output_string
```

Comment:
Do we need/want the wordlist to be case-insensitive? (I have used the capitalization to help me play/cheat at this game in the past haha, just wondering if it is intentional to get rid of that)

Reply:
I can make this function case sensitive if we would like to keep it, I just stuck with the "convert to base" philosophy

Reply:
We need/want both: a list that is all lowercase (if lowercase exists in that alphabet), and a list with the original casing. We can make do with one or the other as a start though and make a separate ticket for the other if that helps.

Reply:
I have added 4 lists, splitChars, splitCharsBase, splitWords and splitWordsBase to cover the 4 cases